src/index.md

```diff
@@ -1,8 +1,10 @@
 # Welcome to Deis Workflow
 
-Deis (pronounced DAY-iss) Workflow is an open source Platform as a Service (PaaS) that adds a developer-friendly layer to any Kubernetes cluster, making it easy to deploy and manage applications on your own servers.  It includes capabilities for building and deploying from source via git push, configuring your applications, managing and rolling back releases, managing application domain names and SSL certificates, edge routing, application log aggregation, and managing access to application resources.  All of this is exposed through a simple REST API and command line interface.
+Deis (pronounced DAY-iss) Workflow is an open source Platform as a Service (PaaS) that adds a developer-friendly layer to any Kubernetes cluster, making it easy to deploy and manage applications on your own servers.
 
-For more detailed information about Workflow view the [Concepts][concepts] and [Architecture][arch] sections.
+Deis Workflow includes capabilities for building and deploying from source via `git push`, configuring your applications, creating and rolling back releases, managing application domain names and SSL certificates, providing edge routing, aggregating application logs, and controlling access to application resources.  All of this is exposed through a simple REST API and command line interface.
```

Comment:
In the docs I have edited, the psuedo-rule I have stuck to is that for the first mention of the product on any given page, I refer to it more formally as "Deis Workflow," and thereafter, more casually as just "Workflow."  Idk it that makes sense to people or seems useful, but it's what I've been observing.